src/components/reusable/breadcrumbs/breadcrumbItem.scss

```
@@ -0,0 +1,31 @@
+@use '../../../common/scss/global.scss';
+
+.breadcrumb-item {
+  display: inline;
+  color: var(--kd-color-text-primary);
+  font-weight: var(--kd-font-weight-medium);
+
+  span,
+  a {
+    padding: 8px;
+  }
+
+  a {
+    text-decoration: none;
+    color: var(--kd-color-text-tertiary);
+    white-space: nowrap;
+    border-radius: 4px;
+    outline: none;
+    border: 2px solid transparent;
+    transition: border-color 0.2s ease-in-out, border-radius 0.2s ease-in-out;
```

Could add a color transition here too for extra polish.

---

I'm not sure which color since border color is the only thing that changes here. Would you please add a suggestion here such as
```suggestion
    transition: border-color 0.2s ease-in-out, border-radius 0.2s ease-in-out, background-color 0.2 ease-in-out;
``` ?

---

Text color changes on hover

---

Ah that, ok.